assets/src/edit-story/components/panels/design/preset/colorPreset/colorPreset.js

```diff
@@ -29,169 +27,12 @@ import { __ } from '@wordpress/i18n';
-    position: absolute;
-    top: calc(50% - ${REMOVE_ICON_SIZE / 2}px);
-    left: calc(50% - ${REMOVE_ICON_SIZE / 2}px);
-    transform: rotate(45deg);
-  }
-`;
-const Color = styled.button.attrs({ type: 'button' })`
-  ${presetCSS}
-  ${({ color }) => generatePatternStyles(color)}
-  transform: rotate(-45deg);
-  &:focus {
-    outline: none !important;
-  }
-
-  svg {
-    color: ${({ theme }) => theme.colors.fg.primary};
-  }
-`;
-
-const OpaqueColorWrapper = styled.div`
-  height: ${PRESET_SIZE}px;
-  width: 50%;
-  overflow: hidden;
-  position: absolute;
-  left: 0;
-  top: 0;
-`;
-
-const OpaqueColor = styled.div`
-  height: ${PRESET_SIZE}px;
-  width: ${PRESET_SIZE}px;
-  position: absolute;
-  top: 0;
-  left: 0;
-  ${({ color }) => generatePatternStyles(color)}
-`;
 
 function ColorPresetPanel({ pushUpdate }) {
-  const { currentPage, selectedElements } = useStory(
-    ({ state: { currentPage, selectedElements } }) => {
-      return {
-        currentPage,
-        selectedElements,
-      };
-    }
-  );
-
-  const isText = areAllType('text', selectedElements);
-  const isBackground = selectedElements[0].id === currentPage.elements[0].id;
-  const colorPresetRenderer = (
-    color,
-    i,
-    activeIndex,
-    handleOnClick,
-    isEditMode,
-    isLocal
-  ) => {
-    if (!color) {
-      return null;
-    }
-    const deleteLabel = isLocal
-      ? __('Delete local color', 'web-stories')
-      : __('Delete global color', 'web-stories');
-    const applyLabel = isLocal
-      ? __('Apply local color', 'web-stories')
-      : __('Apply global color', 'web-stories');
-    const hasTransparency = presetHasOpacity(color);
-    const hasGradient = presetHasGradient(color);
-    const opaqueColor = hasTransparency ? getOpaqueColor(color) : color;
-    // In edit mode we always enable the button for being able to delete.
-    const disabled =
-      !isEditMode &&
-      ((isBackground && hasTransparency) || (isText && hasGradient));
-    let tooltip = null;
-    if (disabled) {
-      tooltip = isBackground
-        ? __('Page background colors cannot have an opacity.', 'web-stories')
-        : __('Gradient not allowed for Text', 'web-stories');
-    }
-    return (
-      <WithTooltip title={tooltip}>
-        <ColorWrapper disabled={disabled}>
-          {hasTransparency && <Transparent />}
-          <Color
-            tabIndex={activeIndex === i ? 0 : -1}
-            color={color}
-            onClick={() => handleOnClick(color)}
-            disabled={disabled}
-            aria-label={isEditMode ? deleteLabel : applyLabel}
-          >
-            {hasTransparency && !hasGradient && (
-              <OpaqueColorWrapper>
-                <OpaqueColor color={opaqueColor} />
-              </OpaqueColorWrapper>
-            )}
-            {isEditMode && <Remove />}
-          </Color>
-        </ColorWrapper>
-      </WithTooltip>
-    );
-  };
-
   return (
     <PresetPanel
       title={__('Saved colors', 'web-stories')}
```

Comment:
I do think that the `type="color"` argument should be explicit here. There's no real reason for color being default over style.

Reply:
It's more likely to be `color` 😛 Ok, will add.